Formula/fisherman.rb

```diff
@@ -0,0 +1,18 @@
+class Fisherman < Formula
+  desc "fish plugin manager"
+  homepage "https://github.com/fisherman/fisherman"
+
+  url "https://raw.githubusercontent.com/fisherman/fisherman/2.12.0/fisher.fish"
+  sha256 "3fa4c7c9a222dca46f7d279989ceb2fff1cc0f30e4f2815bbcd5afe7f8bc3d1e"
+
+  head "https://github.com/fisherman/fisherman.git"
+
+  depends_on "fish"
+
+  def install
+    (share/"fish/vendor_functions.d/").install "fisher.fish"
```

Comment:
Can you explain this install method? It doesn't seem to match anything in the upstream documentation.